docs/source/installing_in_virtualenv.rst

```diff
@@ -0,0 +1,165 @@
+Installing Nitrate on RHEL7 (virtualenv) with Apache and MySQL
```

Comment:
No. In RHEL, RPM is the only recommended way to install Nitrate. virtualenv is another choice to deloy Nitrate in whatever RHEL, CentOS, or Debian, but it's not recommended officially and I also don't want users to misunderstand from official documentation. Anyway, your proposed document is much useful as a guide of "Install Nitrate with virtualenv", obviously you could mention that you choose RHEL and the steps should also apply to other distros (that you have done in document).